Rakefile

```diff
@@ -141,6 +141,10 @@ namespace :arm do
     Dir.chdir(File.expand_path('../management', __FILE__))
     gem_folders = Dir['*'].reject{|o| not File.directory?(o)}
     gem_folders.each do |gem|
+      if GEMS_TO_IGNORE.include?gem
```

Comment:
If we're deleting insights from our generation, do we still need the "GEMS_TO_IGNORE" flag?